README.md

```diff
@@ -89,4 +89,11 @@ If you prefer the command line way of downloading and installing things, then fe
 ## Contribute
 
 -   ECOHABIT appreciates your contribution in any aspects of project development like documenting, UI/UX design, Frontend/Backend development etc.
+-   Please try and follow below guidelines while creating a new PR:
+	- Keep the PR size smaller(max 8 files).
```

Comment:
yes actually initially I was thinking to add LOC but it seemed much more difficult to measure the right quantity. Also, it could be exceptions where more than 8 files are req to be touched for a single feature/fix.

This 8-file limit could force users to break down the feature/bug into smaller units rather than picking a huge task, and the ultimate motive which is to keep the MR smaller so it's easier to review.

Reply:
cool. thanks.